### DISCUSSION TITLE
How to remove remote branch from cloned repo in git.

### DISCUSSION THREAD

**Discussion by hello-work** [community/community#28658] [Other Feature Feedback, Questions, & Ideas] [answered]
How to remove remote repo link to the cloned repo in git.

**Comment by thilina4321** [comment, marked as answer]
- git remote remove origin ( to remove )
- git remote -v ( for check )

**Reply by sudharaD** [reply]
Also to remove it from locally we can use below command as well

`rm -rf .git`

**Reply by airtower-luna** [reply]
Don't do what @sudharaD wrote, it would *completely destroy* your local repository. :warning:

**Reply by sudharaD** [reply]
Hi @airtower-luna . In my comment I already have mentioned about what you have mentioned in above comment. Please read it again.

**Reply by airtower-luna** [reply]
What you wrote will not remove a *remote*, it will delete *all* Git data (history, branches, *everything*).

**Reply by sudharaD** [reply]
Yah that's correct but in my comment I have mentioned it

**Reply by airtower-luna** [reply]
Nothing in your comment describes *what* will be deleted. Of course it's obvious if you understand what `rm -rf` does and what `.git` is, but a beginner who finds this thread might not know that and lose their data.

**Reply by sudharaD** [reply]
Oh sorry if it does not clear. But I have mentioned it will remove the things locally as well. All good
